com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/transaction/ZigBeeTransactionQueue.java

```diff
@@ -161,6 +168,15 @@ public boolean isSleepy() {
         return sleepy;
     }
 
+    /**
+     * Gets the {@link IeeeAddress} of the device for which this queue has transactions
+     *
+     * @return the IEEE address
```

Comment:
It might pay to add that this may be null if the queue is a generic queue not linked to a device?